pkg/resources/config/crd/bases/camel.apache.org_builds.yaml

```diff
@@ -15,12 +15,14 @@
 # limitations under the License.
 # ---------------------------------------------------------------------------
 
+
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.15.0
+    controller-gen.kubebuilder.io/version: v0.6.1
```

Comment:
I think the problem is the controller gen version used. You may need to upgrade to a more recent one.

Reply:
@squakez I'm running `make generate` while the version is 0.15.0 but seems to be getting overridden somewhere after the generate command completes. I feel like I'm still missing something but not sure what yet. I synced my code and resolved conflicts too. Is there a new method that's being used for code generation?